.github/workflows/lint-bpf-checks.yaml

```diff
@@ -42,6 +43,9 @@ jobs:
               - 'contrib/coccinelle/**'
             bpf-tests-runner:
               - 'test/bpf_tests/**'
+              - 'pkg/bpf/**'
```

Comment:
This line is not in the original commit. Please add a note in the commit description.

Reply:
Oups! Removed.